experimental/dds/tree2/src/core/tree/persistedTreeTextFormat.ts

```diff
@@ -79,7 +79,7 @@ interface EncodedGenericTreeNode<TChild>
 	extends EncodedGenericFieldsNode<TChild>,
 		EncodedNodeData {}
 const EncodedGenericTreeNode = <Schema extends TSchema>(tChild: Schema) =>
-	Type.Intersect([EncodedGenericFieldsNode(tChild), EncodedNodeData], {
+	Type.Composite([EncodedGenericFieldsNode(tChild), EncodedNodeData], {
```

Comment:
I'm not sure why typebox's validator tolerated this before, but when using ajv this encoding complained on some optional field tests.

In general, `Type.Intersect` is a bad idea to use with `{ additionalProperties: false }`. The difference between the two is that `Type.Composite` constructs a single object type with the necessary fields, whereas `Type.Intersect` uses JSON schema's `allOf` specification.